src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,34 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    public static final int INT = 11;
+    public static final int INT1 = -1;
+    public static final int twelve = 12;
```

Comment:
Names do not explain anything. 
Wrong convention (twelve).
Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz